lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2417,6 +2417,13 @@ mapping:
           Details" option in the history. This option also exposes the command line to
           non-administrative users. Administrators can always see dataset paths.
 
+      job_metrics_config_file:
+        type: str
+        default: config/job_metrics_conf.xml
```

Comment:
```suggestion
        default: job_metrics_conf.xml
```

Reply:
It will be resolved w.r.t. `config_dir` here (regardless of what `config_dir` is set to): https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L765